File: Tarea07/ejercicio1.py
f = [0,0,0,0,0,0,0,0,0,0]
enc = 0
print("Ingresa 10 numeros")
for n in range(0, 10):
	f[n] = int(input())
print("Ingresa el numero a buscar")
num1 = int(input())
for n in range(0, 10):
	if f[n] == num1:
		enc = 1
if enc == 1:
	print("El numero %d esta en la lista"%num1)
else:
	print("El numero %d no esta en la lista"%num1)

File: Tarea07/ejercicio1.c
#include <stdio.h>

int main(){
	int n, f[10], num1, enc;
	printf("Ingersa 10 numeros\n");
	for (n = 0; n < 10; ++n){
		scanf("%d", &f[n]);
	}
	printf("Ingersa el numero a buscar\n");
	scanf("%d", &num1);
	for (n = 0; n < 10; ++n){
		if (f[n]==num1){
			enc = 1;
		}
	}
	if (enc == 1){
		printf("El numero %d esta en la lista\n", num1);
	}
	else{
		printf("El número %d no esta en la lista\n", num1);
	}
	return 0;
}